plugins/woocommerce/src/Blocks/BlockTypes/ProductFilterPrice.php

```diff
@@ -127,15 +127,16 @@ protected function render( $attributes, $content, $block ) {
 		$max_price   = intval( get_query_var( self::MAX_PRICE_QUERY_VAR, $max_range ) );
 
 		$filter_context = array(
-			'price'   => array(
+			'price'           => array(
 				'minPrice' => $min_price,
 				'maxPrice' => $max_price,
 				'minRange' => $min_range,
 				'maxRange' => $max_range,
 			),
-			'actions' => array(
+			'actions'         => array(
 				'setPrices' => "{$this->get_full_block_name()}::actions.setPrices",
 			),
+			'filterParamKeys' => array( self::MIN_PRICE_QUERY_VAR, self::MAX_PRICE_QUERY_VAR ),
```

Comment:
Still on the fence about `filterParamKey` vs `filterParamKeys`. The price filter is the only one that needs multiple keys.

Reply:
I think the plural form is fine.